docs/developer/attributes.mdx

```diff
@@ -5,33 +5,33 @@ sidebar_label: Attributes
 
 ## Introduction
 
-Setting up attributes will allow your users to filter your products catalogue.
+Properly configured attributes provide crucial information about your products and allow users to find the ones they are looking for quicker.
 
 Attributes can be used for
 - [Products](developer/api-reference/objects/product.mdx) and [ProductVariants](developer/api-reference/objects/product-variant.mdx)
 - [Pages](developer/api-reference/objects/page.mdx)
 
 ## Use cases
 
-1. Your store sells Mugs in many colors and sizes. This means you will need:
+1. Your store sells mugs in many colors and sizes. In this example, you can configure the following attributes:
 - Color attribute (red, black, etc.)
-- Size attribute (Small, Big)
+- Size attribute (small, big)
 
-2. Your shop has many Pages which should render with different styles. To achieve that, you can create attribute named `Template` with type `PAGE_TYPE`. As values you could set: `Support`, `Blog`, `Promos`. Then in storefront code you could load proper css template based on page `Template` value.
+2. Your shop has many pages with different rendering styles. You can create an attribute called `Template` with a `PAGE_TYPE` type. You can set the following values: `Support`, `Blog`, `Promos`. Then you can upload a proper CSS template based on a `Template` value in the storefront code. 
 
 ## Permissions
 
-To edit and create Attributes user is required to have permissions:
+To edit and create attributes, a user needs to have the following permissions:
 - `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES`
 - `MANAGE_PAGE_TYPES_AND_ATTRIBUTES`
 
-Assigning Attrubutes to Products and Pages does not require additional permission besides `MANAGE_PRODUCTS` and `MANAGE_PAGES`
+To assign attributes to products and pages, a user needs only the following permissions: `MANAGE_PRODUCTS` and `MANAGE_PAGES`.
 
 ## Filtering by attributes
 
-For filtering by attributes you can use their `slug` name. This way you can be sure that query parameters can be encoded in the URL and will be the same in all languages.
+A user can use a `slug`  to filter products by attributes. The query parameters can be encoded in URL and they will be the same in all languages.
 
-Example of [`products`](developer/api-reference/queries/products.mdx) query:
+An example of [`products`](developer/api-reference/queries/products.mdx) query:
```

Comment:
An example of a [`product's`](developer/api-reference/queries/products.mdx) query: